Mathlib/Tactic/Polyrith.lean

```diff
@@ -315,7 +315,7 @@ to replace the call to `polyrith` with the appropriate call to
 This returns `none` if this was a "dry run" attempt that does not actually invoke sage.
```

Comment:
This docstring is no longer accurate

Reply:
@eric-wieser I'll try to address your concern to save Mario some time, but what part of the docstring is not longer accurate in your opinion?

Reply:
It looks too me like it no longer rerurns `none` as the return type is no longer `Option`.

Reply:
```suggestion
Returns `.error g` if this was a "dry run" attempt that does not actually invoke sage.
```